plugin/pkg/auth/authorizer/node/node_authorizer.go

```diff
@@ -98,17 +102,42 @@ func (r *NodeAuthorizer) Authorize(attrs authorizer.Attributes) (authorizer.Deci
 	return authorizer.DecisionNoOpinion, "", nil
 }
 
+// authorizeStatusUpdate authorizes get/update/patch requests to status subresources of the specified type if they are related to the specified node
+func (r *NodeAuthorizer) authorizeStatusUpdate(nodeName string, startingType vertexType, attrs authorizer.Attributes) (authorizer.Decision, string, error) {
```

Comment:
should deny if feature is off

Reply:
the feature gate check probably belongs in
```
case pvcResource:
  if attrs.GetSubresource() == "status" {
    ... check feature gate ...
    ...
```

Reply:
fixed